SPEC.md

```diff
@@ -118,6 +118,13 @@ Image Format TODO
 * Define security requirements for a container. In particular is any isolation of users required between containers? What user does each application run under and can this be root (i.e. "real" root in the host).
 * Define how apps are supposed to communicate; can they/do they 'see' each other (a section in the apps perspective would help)?
 
+### Pod Manifest
```

Comment:
Propose not using "pod". A group of colocated apps in one container is not the same model as the kubernetes "pod" (a group of containers). I don't like the idea of using a borrowed word from something that is actually a bit different.